pysages/methods/ffs.py

```diff
@@ -263,13 +269,15 @@ def basin_sampling(
             print("Restoring basing configuration since system leave basin with cv value:\n")
```

Comment:
...system leave basin... --> ...system **left** basin... ?
@gustavor101
but I'm not sure since I'm not so familiar with the method.

Reply:
yeah, I think you are right here @ricalessandri